api/CSSPageDescriptors.json

```diff
@@ -0,0 +1,389 @@
+{
+  "api": {
+    "CSSPageDescriptors": {
+      "__compat": {
+        "spec_url": "https://drafts.csswg.org/cssom/#csspagedescriptors",
+        "support": {
+          "chrome": {
+            "version_added": false
+          },
+          "chrome_android": "mirror",
+          "edge": "mirror",
+          "firefox": {
+            "version_added": "129"
+          },
+          "firefox_android": {
+            "version_added": false
+          },
```

Comment:
It seems strange for this interface to have a difference of support between desktop and mobile, and the collector should be setting this to mirror by default...

Reply:
It wasn't a manual change. This comes from directly from running the collector. I agree it should have been mirror by default.

Reply:
Ah, wait, I think I know what's going on!  It's because tests are only run for Firefox Desktop beta, but we have results for Firefox Android stable as well.  The collector sees the Firefox Android results file and sees that the feature is not supported, and therefore sets it to `false` accordingly.  The collector has code to detect if BCD indicates support was added in a later version than what we have results for, but it's not smart enough to handle mirrored results.

Reply:
Updated to mirror.